src/components/dialogs/network-modifications/voltage-level-topology-modification/voltage-level-topology.type.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) 2025, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+import { EquipmentModificationDialogProps } from '../../../graph/menus/network-modifications/network-modification-menu.type';
+import { TopologyVoltageLevelModificationInfos } from '../../../../services/network-modification-types';
+import { CURRENT_CONNECTION_STATUS, PREV_CONNECTION_STATUS, SWITCH_ID } from '../../../utils/field-constants';
+import { CustomHeaderProps } from 'ag-grid-react';
+import React, { ComponentType } from 'react';
+
+export type SwitchInfos = {
+    id: string;
+    open: boolean;
+};
+
+export type VoltageLevelTopologyModificationDialogProps = EquipmentModificationDialogProps & {
+    editData: TopologyVoltageLevelModificationInfos;
+};
+
+export interface SwitchRowData {
+    [SWITCH_ID]: string;
+    [PREV_CONNECTION_STATUS]: string;
+    [CURRENT_CONNECTION_STATUS]: boolean | null;
+    isModified?: boolean;
+    isSeparator?: boolean;
+}
+
+export interface CellStyleParams {
```

Comment:
```suggestion
export interface SwitchCellStyleParams {
```
Yours name is so generic while you are typing for switch data

Reply:
done